cargo-dist/src/errors.rs

```diff
@@ -626,6 +626,10 @@ pub enum DistError {
         /// Version the project uses
         your_version: semver::Version,
     },
+
+    /// No dist manifest
+    #[error("No configuration file (e.g. dist-workspace.toml) was found")]
```

Comment:
we may want to give some more help here- could people on v0 who havent yet migrated get this?

Reply:
people on v0 can't get this -- it's only returned by `dist::v1::loader::*`, which is only used if we explicitly do not have a v0 config.

Reply:
added help text